tests/scripts/analyze_outcomes.py

```diff
@@ -22,17 +22,20 @@ def __init__(self):
         self.error_count = 0
         self.warning_count = 0
 
-    @staticmethod
-    def log(fmt, *args, **kwargs):
-        sys.stderr.write((fmt + '\n').format(*args, **kwargs))
+    def info(self, fmt, *args, **kwargs):
+        self._print_line('Info: ' + fmt, *args, **kwargs)
 
     def error(self, fmt, *args, **kwargs):
-        self.log('Error: ' + fmt, *args, **kwargs)
         self.error_count += 1
+        self._print_line('Error: ' + fmt, *args, **kwargs)
 
     def warning(self, fmt, *args, **kwargs):
-        self.log('Warning: ' + fmt, *args, **kwargs)
         self.warning_count += 1
+        self._print_line('Warning: ' + fmt, *args, **kwargs)
+
+    @staticmethod
+    def _print_line(fmt, *args, **kwargs):
+        sys.stderr.write(fmt + '\n', *args, **kwargs)
```

Comment:
[The CI is unhappy.](https://mbedtls.trustedfirmware.org/job/mbed-tls-pr-head/job/PR-8335-head/6/execution/node/7964/log/) You need to restore the call to `format`.

```suggestion
        sys.stderr.write((fmt + '\n').format(*args, **kwargs))
```